lib/avrora/encoder.ex

```diff
@@ -16,6 +16,32 @@ defmodule Avrora.Encoder do
     map_type: :proplist,
     record_type: :map
   }
+  @ocf_schema_full_name_pos 6
```

Comment:
Let's use an existing name pattern `@object_container_` and move it closer to another OCF variable on top